test/store.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+function b64_decode() {
+  base64 -d "$@"
+}
+if base64 -D <(echo "Cg==") &> /dev/null || false; then
```

Comment:
What's happening here? You're redefining the method?

Reply:
Yup. Depending on where this is running the flag is `-D` or `-d` so we redefine if `-D` works.